apps/dash-nlp/ldacomplaints.py

```diff
@@ -102,7 +110,13 @@ def tsne_analysis(ldamodel, corpus):
     topic_nums = np.argmax(df_topics, axis=1)
 
     # tSNE Dimension Reduction
-    tsne_model = TSNE(n_components=2, verbose=1, random_state=0, angle=0.99, init="pca")
-    tsne_lda = tsne_model.fit_transform(df_topics)
+    try:
+        tsne_model = TSNE(
+            n_components=2, verbose=1, random_state=0, angle=0.99, init="pca"
+        )
+        tsne_lda = tsne_model.fit_transform(df_topics)
+    except:
+        print("TSNE_ANALYSIS WENT HORRIBLY WRONG")
```

Comment:
@ycaokris If it's possible to return the error code or other useful debugging information here, that would probably help in case anyone runs into this message. Just a suggestion.

Reply:
@lundstrj  this could be more informative